### ISSUE TITLE
ERROR toggleRenderControlCharacters

### ISSUE THREAD

**Issue by ZCShou** [Microsoft/vscode#8823]
- VSCode Version: Code - Insiders 1.3.0-insider (e724f269ded347b49fcf1657fc576399354e6703, 2016-07-06T13:46:20.448Z)
- OS Version: Windows_NT ia32 10.0.10586

Steps to Reproduce:

1. menu->view->toggleRenderControlCharacters
![2016-07-07_080127](https://cloud.githubusercontent.com/assets/8591065/16638441/e265a3f6-4419-11e6-865d-fc3789c6b7e2.png)

**Comment by alexandrudima** [comment]
The menu item was introduced by PR #7578 but there is no editor action backing up the menu item.

**Comment by f111fei** [comment]
@alexandrudima  https://github.com/Microsoft/vscode/pull/7578/files#diff-2907cb78eb352fd52d5e0e4f1d090d8aR172

**Comment by alexandrudima** [comment]
Oh, yes you are correct. It is a simple typo.
```
editor.action.toggleRenderControlCharacters (in menu item)
editor.action.toggleRenderControlCharacter (in action)
```

@bpasero Is there any way for us to avoid such issues?

**Comment by bpasero** [comment]
@alexandrudima maybe when we send the commands to the workbench to resolve keybindings we could also check if the command exists and log a warning otherwise (https://github.com/Microsoft/vscode/blob/master/src/vs/code/electron-main/menus.ts#L135)

**Comment by alexandrudima** [comment]
@bpasero Good idea - also found broken `workbench.action.files.openFileFolder`

**Comment by bpasero** [comment]
@alexandrudima careful though, some commands are platform dependent. that command only runs on Mac!

**Comment by jrieken** [comment]
removing candidate tag after discusion

**Comment by alexandrudima** [comment]
The fix is in master.

**Comment by jd-carroll** [comment]
Was this supposed to be fixed in 1.3.1?

**Comment by Tyriar** [comment]
@jd-carroll It's set to the July milestone so that's 1.4.0.
